web/models/attendance/attendance.ex

```diff
@@ -14,9 +17,32 @@ defmodule CoursePlanner.Attendance do
 
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:attendance_type, :class_id, :student_id])
+    |> cast(params, [:attendance_type, :class_id, :student_id, :comment])
     |> validate_required([:attendance_type, :class_id, :student_id])
     |> cast_assoc(:class)
     |> cast_assoc(:student)
+    |> validate_comment()
```

Comment:
yes, it will look like this 
`
struct
    |> cast(params, [:attendance_type, :class_id, :student_id, :comment])
    |> validate_required([:attendance_type, :class_id, :student_id])
    |> cast_assoc(:class)
    |> cast_assoc(:student)
    |> validate_inclusion(:comment, Settings.get_value("ATTENDANCE_DESCRIPTORS"))`

I found it ugly and that's why i wrote another function, but if it's good I change it to this one